spyderlib/spyder.py

```diff
@@ -2743,6 +2743,9 @@ def restart(self):
             startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
             shell = False
         else:
+            # This environment variable is needed so that the new python
+            # process spawned with Popen sets this as the default encoding
+            env['PYTHONIOENCODING'] = 'utf-8'

```

Comment:
I think setting this system-wide is not a good idea. For example, It could mess with Pandas trying to import/export files :-)